gradle/platform/build.gradle.kts

```
@@ -0,0 +1,42 @@
+plugins {
+    id("java-platform")
+}
+
+group = "com.kotlin-language-server"
```

This should ideally match up with the package structure or at least an actual domain. Since the packages live in `org.javacs.kt` (as inherited by the original language server repo), we could use that with the disadvantage of it not being very descriptive.

As an alternative, I would propose

```suggestion
group = "dev.fwcd.kotlin-language-server"
```

Whether we would want to move the actual package structure to that too would be a topic for future debate.

---

Sure thing, I've made the adjustment in my next revision!